recipes-devtools/upm/upm_0.7.1.bb

```diff
@@ -16,16 +16,14 @@ inherit distutils-base pkgconfig python-dir cmake
 CFLAGS_append_edison = " -msse3 -mfpmath=sse"
 
 FILES_${PN}-doc += " ${datadir}/upm/examples/"
-RDEPENDS_${PN} += " mraa"
+RDEPENDS_${PN} += " mraa mraa-java"
```

Comment:
We are splitting them out into different packages.  The above line is wrong because mraa-java is an optional runtime-dependency.